src/backend/Models.ts

```diff
@@ -69,6 +69,13 @@ const postSchema = new Schema({
 });
 export const Post = model('post', postSchema);
 
+export interface databaseChat extends Document{
```

Comment:
Do you think interfaces for models should be shared with frontend so parsing is easier and more readable?